distribution/changelog.txt

```diff
@@ -1,5 +1,6 @@
 0.3.5+ (in development)
 ------------------------------------------------------------------------
+- Fix: [#15844] Tile Inspector inconsistent text colour. 
```

Comment:
Move this down to the end of the list. The changelog is ordered by change type, then by issue ID.

Reply:
Got it.